modal/_function_utils.py

```diff
@@ -97,9 +96,6 @@ def __init__(self, f, serialized=False):
             self.is_package = False
             self.is_file = True
         else:
-            # Use cloudpickle. Used when working w/ Jupyter notebooks.
```

Comment:
Serialization was moved from decoration time to _load time in order to support class serialization at the same time